pkg/capabilities/capabilities_test.go

```diff
@@ -48,10 +42,57 @@ func TestGet(t *testing.T) {
 			HostNetworkSources: []string{"A", "B"},
 		},
 	}
-	SetForTests(cap)
+	ResetForTest(&cap)
```

Comment:
Here we call `ResetForTest` twice - before the test and after it. I think calling it after the test may be redundant.

Reply:
You're right, the call to ResetForTest after the test is indeed redundant, I'll take it out!

Reply:
/lgtm